client/src/pages/Home.css

```diff
@@ -9,3 +9,77 @@
 .message:hover {
 	background-color: yellow;
 }
+
+/* ------------------------ Footer Component ------------------ */
+
+/* -- medeia query mobile, tablet and desktop */
+/* and waiting the  shima's PR  to linke footer the the new pages route*/
+
+.card-footer {
+	background-color: rgb(27, 4, 65);
```

Comment:
Hi Khadar, good job.
I notice there is a difference in the background colour from the one on the users story [https://user-images.githubusercontent.com/102039379/223477432-54fb6d47-0ba4-46b1-80f1-50ac115277f3.png](url)